landoui/assets_src/css/components/FlashMessages.scss

```diff
@@ -0,0 +1,21 @@
+@import "../colors";
+
+.FlashMessages {
+  padding: 1.5rem;
+
+  &-content {
+    margin-bottom: 1rem;
```

Comment:
Nit pick: This combination really pushes the content of the page down a lot. I think we could reduce the amount of padding on the top and bottom and the bottom margin a bit. Up to you.

Reply:
@purelogiq Done.
![Screenshot_2019-12-17 D4 - Lando - Mozilla](https://user-images.githubusercontent.com/2806587/71029207-e419a000-20dc-11ea-8749-91a459289a8d.png)

Reply:
@mars-f I don't see any change between the screenshots or the code. Is that just me?

Reply:
@purelogiq oops, I pushed the update to a branch with the same name on another remote.  This PR is fixed now, have a look again?

Reply:
LGTM!